.gitignore

```diff
@@ -2,3 +2,4 @@
 /telegraf
 /telegraf.exe
 /telegraf.gz
+/.idea
```

Comment:
Remove this, but you can put it in your global excludes file.  If you don't have one just do something like this:
```
git config --global core.excludesfile /home/ertaoxu/.gitignore
cat "/.idea" >> /home/ertaoxu/.gitignore
```